src/index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="fr">
+    <!-- Lire le code de ce genre d’outils est une très bonne précaution 👍. -->
+    <!-- https://github.com/gip-inclusion/calcul-nir/ -->
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="Utilitaire de calcul de la clé NIR (numéro de sécurité sociale) à partir des 13 chiffres le composant.">
+        <meta name="keywords" content="clé,nir,numéro sécurité sociale">
+        <title>Calcul clé NIR</title>
```

Comment:
```suggestion
        <title>Calcul de clé NIR</title>
```
Pour coller au titre de la page ?

Reply:
Oui, c’est aussi plus français :+1: